ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -322,12 +317,17 @@ TensorFilterLlamaCpp::generateTokens (GstTensorFilterProperties *prop,
       throw std::invalid_argument ("Failed to convert token to piece");
     }
 
-    output_async = createOutputTensor (buf, n, prop);
-    if (!output_async)
-      throw std::runtime_error ("generateTokens: Failed to create output tensor");
+    if (prop->invoke_async) {
+      output_async = g_new0 (GstTensorMemory, 1);
+      createOutputTensor (std::string (buf, n), output_async, prop);
+      if (!output_async)
```

Comment:
createOutputTensor() after checking allocation failure.

Reply:
Thank you, I've updated it.